tsconfig.common.json

```diff
@@ -15,6 +15,7 @@
     ],
     "moduleResolution": "node",
     "esModuleInterop": false,
+    "resolveJsonModule": true,
```

Comment:
The vsce bundled script needs to read the package.json file when bundling extensions.  DIdn't hurt anything to turn this on so I updated it.  It allows us to import json files in typescript.